Simplenote/SPAppDelegate+Extensions.swift

```diff
@@ -22,9 +25,13 @@ extension SPAppDelegate {
         simperium.authenticationShouldBeEmbeddedInNavigationController = true
         simperium.delegate = self
 
-        let buckets = [Note.self, Tag.self, Settings.self, Preferences.self].compactMap { entityType in
-            simperium.bucket(forName: entityType.classNameWithoutNamespaces)
-        }
+        let buckets = [
+            simperium.accountBucket,
+            simperium.notesBucket,
+            simperium.preferencesBucket,
+            simperium.settingsBucket,
+            simperium.tagsBucket,
+        ]
```

Comment:
Maybe we can move this to `Simperium+Simplenote` and call it `allBuckets`?

Reply:
Thank youuu